app/upgrades/v4/upgrades.go

```diff
@@ -23,6 +23,8 @@ func CreateUpgradeHandler(
 			return vm, err
 		}
 
+		// TODO: add gov migrations.
```

Comment:
Can you detail the migration needed here plz?

Reply:
https://github.com/atomone-hub/atomone/pull/248/commits/4d840f5c300ed27810309de9966e80a80ded1cdf